service/worker/scanner/scanner.go

```diff
@@ -49,7 +49,6 @@ const (
 )
 
 type (
-	contextKey string
```

Comment:
these are intentionally a different type - e.g. if we ever had a workflow named `processorContext`, they could collide.

Reply:
okay, thanks for your review, I have removed the Common.Context